src/codegen/compile.ml

```diff
@@ -4701,13 +4701,16 @@ module StableMem = struct
     E.add_global64 env "__stablemem_size" Mutable 0L
```

Comment:
Maybe this could be conditional on Flags.use_stable_regions too? Not sure anymore.

Reply:
I'm also a bit confused by how to use the flag correctly.

Today I re-ran all of the tests (post adding this flag logic), and there are failing tests.  I checked in the list into the repo (temporarily) and the script I used to get them (also just a temp file).

The tests that use the "custom max" flag fail.

The tests that do sub-word sized operations (nat8, int8, nat16 and int16) also each fail.

Any ideas about what's going wrong?

Reply:
You mean --max-stable-pages? I think that limits the growth of _stable_mem_size so perhaps worth investigating.